app/controllers/api/threads_controller.rb

```diff
@@ -47,7 +47,9 @@ def unpin
   def create_params
     subforum = Subforum.find(params[:subforum_id])
     params.require(:thread).permit(:title).
-      merge(created_by: current_user, subforum: subforum)
+      merge(created_by: current_user,
+        last_post_created_by: current_user,
```

Comment:
This change was required by enabling `config.active_record.belongs_to_required_by_default`. We set `last_post_created_by` when we create the first post, but we always know who is creating the thread, so I decided to do this rather than setting `optional: true` on the association.